src/threading/Manager.cc

```diff
@@ -140,16 +140,16 @@ void Manager::Terminate() {
 
     // Signal all to stop.
 
-    for ( all_thread_list::iterator i = all_threads.begin(); i != all_threads.end(); i++ )
-        (*i)->SignalStop();
+    for ( const auto& t : all_threads )
```

Comment:
`const auto*` - here and elsewhere in this file as these are all raw pointers, the `&` is confusing.